packages/types/src/project.ts

```diff
@@ -258,7 +258,7 @@ export type SecondLayerHandlerProcessor<
   | SecondLayerHandlerProcessor_0_0_0<EthereumRuntimeHandlerInputMap, K, F, E, DS, ApiWrapper>
   | SecondLayerHandlerProcessor_1_0_0<EthereumRuntimeHandlerInputMap, K, F, E, DS, ApiWrapper>;
 
-type SecondLayerHandlerProcessorArray<
+export type SecondLayerHandlerProcessorArray<
```

Comment:
I think the fix from https://github.com/subquery/subql/pull/2469 also needs to be applied to this file. `EthereumRuntimeHandlerInputMap` -> `EthereumRuntimeFilterMap`